### ISSUE TITLE
Apparently dependency on C++11 being available

### ISSUE THREAD

**Issue by quantum5** [DMOJ/judge#3]
```pytb
Traceback (most recent call last):
  File "/var/lib/openshift/532715544382ec40ff00017a/app-root/data/judge/judge.py", line 269, in _begin_grading
    short_circuit=short_circuit, source_code=original_source, interactive=('grader' in init_data)):
  File "/var/lib/openshift/532715544382ec40ff00017a/app-root/data/judge/judge.py", line 566, in run
    topen = self._resolve_open_call(init_data, problem_id, forward_test_cases)
  File "/var/lib/openshift/532715544382ec40ff00017a/app-root/data/judge/judge.py", line 356, in _resolve_open_call
    generator_launcher = clazz.Executor('%s-generator' % problem_id, generator_source).launch_unsafe
  File "/var/lib/openshift/532715544382ec40ff00017a/app-root/data/judge/executors/GCCExecutor.py", line 57, in __init__
    raise CompileError(compile_error)
CompileError: cc1plus: error: unrecognized command line option "-std=c++11"
```

**Comment by Xyene** [comment]
Should have a `grader-language` field or similar.

**Comment by FatalEagle** [comment]
Could be fixed.

**Comment by quantum5** [comment]
Fixed in 4fe437954a992ec8cdcaa23f69c97313a8d21df3 ages ago.
